pkg/monitor/nodedetails/node_log.go

```diff
@@ -0,0 +1,66 @@
+package nodedetails
+
+import (
+	"bufio"
+	"bytes"
+	"compress/gzip"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"k8s.io/client-go/kubernetes"
+)
+
+// GetNodeLog returns logs for a particular systemd service on a given node.
+// We're count on these logs to fit into some reasonable memory size.
```

Comment:
Those logs are pretty big. Also, you added the MonitorEventsOptions to each run-test command. I believe a full e2e suite might spawn lots of run-tests. Will that be a problem?